.circleci/config.yml

```diff
@@ -1,82 +1,137 @@
 version: 2
 ## Customize the test machine
 jobs:
-  build:
+  checkout_ilastik:
     working_directory: ~/ilastik/ilastik
     parallelism: 1
-    shell: /bin/bash --login
     environment:
-      CONDA_ROOT: /home/circleci/miniconda
+      BASH_ENV: /opt/conda/z/profile.d/conda.s
+      ILASTIK_ROOT: ~/ilastik
+      ON_CIRCLE_CI: 1
       TEST_ENV_NAME: test-env
-      ILASTIK_ROOT: /home/circleci/ilastik
       VOLUMINA_SHOW_3D_WIDGET: 0
-      ON_CIRCLE_CI: 1
     docker:
-    - image: cimg/base:2020.01
+    - image: continuumio/miniconda3
     steps:
-    # add interpolated environment variables (CircleCI 2.0 does not support direct interpolation)
-    - run: echo 'export PATH=${CONDA_ROOT}/bin:${PATH}' >> $BASH_ENV
     - checkout
-    # Prepare for artifact and test results  collection equivalent to how it was done on 1.0.
-    # In many cases you can simplify this from what is generated here.
-    # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
-    # - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
-    # Restore the dependency cache
+    - run: git clone http://github.com/ilastik/ilastik-meta ${ILASTIK_ROOT}/ilastik-meta
+    - run: >
+        cd ${ILASTIK_ROOT}/ilastik-meta &&
+        git submodule init &&
+        git submodule update --recursive &&
+        git submodule foreach "git checkout master"
     - run: >
-        if [[ ! -d ${CONDA_ROOT} ]]; then
-            echo "Installing Miniconda...";
-            wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh &&
-            bash Miniconda3-latest-Linux-x86_64.sh -b -p ${CONDA_ROOT};
-        else
-            echo "Using cached Miniconda install";
-        fi
-    - run: echo 'source "$CONDA_ROOT/etc/profile.d/conda.sh"' >> $BASH_ENV
+        rm -rf ${ILASTIK_ROOT}/ilastik-meta/ilastik &&
+        ln -s `pwd` ${ILASTIK_ROOT}/ilastik-meta/ilastik
+    - save_cache:
+        key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+        paths:
+        - ~/ilastik
+
+  configure_conda:
+    working_directory: ~/ilastik/ilastik
+    parallelism: 1
+    environment:
+      BASH_ENV: /opt/conda/etc/profile.d/conda.s
```

Comment:
Typo in path should end with .sh

Reply:
I've removed it completely - the miniconda3 container already seems to be correctly configured